### DISCUSSION TITLE
v4.0.0-rc.0

### DISCUSSION THREAD

**Discussion by wisskid** [smarty-php/smarty#679] [General]
Pre-release for Smarty4 support PHP8.0!

<hr /><em>This discussion was created from the release <a href='https://github.com/smarty-php/smarty/releases/tag/v4.0.0-rc.0'>v4.0.0-rc.0</a>.</em>

**Comment by Gemorroj** [comment]
full changelog https://github.com/smarty-php/smarty/blob/v4.0.0-rc.0/CHANGELOG.md#400-rc0---2021-10-13

**Comment by wisskid** [comment]
Smarty 4 has been released
